test/passing/gen/gen.ml

```diff
@@ -72,8 +93,9 @@ let emit_test test_name setup =
 (alias
  (name runtest)
  (action (diff %s %s.output)))
-|}
+%s|}
     test_name extra_deps cmd_prefix opts base_test_name ref_name test_name
+    when_ocp
```

Comment:
Passing that as a string argument is not necessary. This function (`emit_test`) is printing so this could be directly printed to stdout.

Reply:
Indeed, fixed.